src/util/getCssSelectorLocations.js

```diff
@@ -0,0 +1,19 @@
+import postcss from 'postcss';
+
+export default function getCssSelectorLocations(source) {
+  try {
+    const rootNode = postcss.parse(source);
+    const selectors = [];
+
+    rootNode.walkRules((rule) => {
+      selectors.push({
+        selector: rule.selector,
+        loc: rule.source,
+      });
+    });
+
+    return selectors;
+  } catch (e) {
```

Comment:
Should we be looking for a specific error that indicates invalid CSS syntax? I wouldn’t want to inadvertently swallow an error thrown by a bug in our code.